src/cargo/util/toml/mod.rs

```diff
@@ -1538,7 +1537,7 @@ impl schema::InheritableFields {
     }
 
     /// Gets a workspace dependency with the `name`.
-    pub fn get_dependency(
+    fn get_dependency(
```

Comment:
While I understand and agree the purpose of making them private, I am not sure if other projects depends on these and might be unhappy.